test/Microsoft.AspNet.Mvc.FunctionalTests/ViewEngineTests.cs

```diff
@@ -288,7 +292,7 @@ public static IEnumerable<object[]> RazorViewEngine_UsesExpandersForLayoutsData
 View With Layout
 </language-layout>";
 
-                yield return new[] { "gb", expected1 };
+                yield return new[] { "en-GB", expected1 };
```

Comment:
We want it to be locale specific right? For instance date format will be different in `en-US` and `en-GB` but `CultureInfo.TwoLetterISOLanguageName` for both will be `en`